docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     env_file:
     - .env
     volumes:
-    - .:/app
+    - .:/src
```

Comment:
The docker container can't currently write to the filesystem. This fixes that. This should really be it's own PR....